src/types.ts

```diff
@@ -1,5 +1,6 @@
 export type CameraApi = {
   capture: () => Promise<{ uri: string }>,
+  setTorchMode: (mode: String) => void;
```

Comment:
Can you use a Union type string here? Ie the different string allowed, "on", etc

Reply:
yes, i will use a union type string instead.